src/core/generators/query.ts

```diff
@@ -885,7 +886,20 @@ export const generateQuery: ClientBuilder = (
   options,
   outputClient,
 ) => {
-  const imports = generateVerbImports(verbOptions);
+  let imports = [] as GeneratorImport[];
+
```

Comment:
By adding the logic here you only doing it for the query templates. I think it would be better to do it as soon as possible. Also in the next big release this will probably be removed because I want to add a better way to generate the endpoints more customizable